quic/s2n-quic-transport/src/space/mod.rs

```diff
@@ -282,6 +284,56 @@ impl<Config: endpoint::Config> PacketSpaceManager<Config> {
         Poll::Ready(Ok(()))
     }
 
+    #[allow(clippy::too_many_arguments)]
+    pub fn post_handshake_crypto<Pub: event::ConnectionPublisher>(
+        &mut self,
+        path_manager: &mut path::Manager<Config>,
+        local_id_registry: &mut connection::LocalIdRegistry,
+        limits: &mut Limits,
+        now: Timestamp,
+        waker: &Waker,
+        publisher: &mut Pub,
+        datagram: &mut Config::DatagramEndpoint,
+    ) -> Result<(), transport::Error> {
+        if let Some(session_info) = self.session_info.as_mut() {
+            let mut context: SessionContext<Config, Pub> = SessionContext {
+                now,
+                initial_cid: &session_info.initial_cid,
+                retry_cid: self.retry_cid.as_deref(),
+                initial: &mut self.initial,
+                handshake: &mut self.handshake,
+                application: &mut self.application,
+                zero_rtt_crypto: &mut self.zero_rtt_crypto,
+                path_manager,
+                handshake_status: &mut self.handshake_status,
+                local_id_registry,
+                limits,
+                server_name: &mut self.server_name,
+                application_protocol: &mut self.application_protocol,
+                waker,
+                publisher,
+                datagram,
+            };
+
+            let _result = session_info
```

Comment:
Not sure how I missed this but I think we need to handle this error, right? Like what if the ticket message is malformed? I think it's best to close the connection at this point. You should be able to just do `?` at the end of the call.

Reply:
Yeah I wasn't sure what to do in this situation. I guess erroring makes the most sense.